R/style.R

```diff
@@ -243,7 +243,7 @@ m_style_label <- function(
 #'   m_zoom_to()
 #' @export
 m_style_line <- function(
-                         colorScheme = "default",
+                         colorScheme = NULL,
```

Comment:
How about also fix the documentations about `colorScheme`?

```
#' @param colorScheme Specify scheme to color the atoms by. Default is
#' "default". Other choies are "Carbon", ssPyMOL", "ssJmol", "Jmol",
#' "default", "amino",
#' "shapely", "nucleic", "chain", "chainHetatm", "prop".
```

- Maybe there is a typo in `choies`?
- A quotation is missing before `ssPyMOL`